### PR TITLE
Delete resources from frontend app

### PR DESCRIPTION
**Pull request by luiztauffer**
**WIP**

Issue #248 

- [x] trigger callback with `active_cell` prop from dash datatable
- [x] use sageworks client to request delete of artifacts in the aws stack
- [x] update datatable by removing the specific row
- [x] improve the style of "delete" cells, maybe try to add a icon through html embedded in the markdown cell
- [x] add confirmation dialog, modify callbacks accordingly

### CODE REVIEW

**Review by brifordwylie — COMMENTED** [review]
Okay.. so the trash can looks fine.. there's a couple of issues...

1) Have the 'trash can' be an argument/option on the table that can be set to False for the incoming data table

Screenshot 2023-06-06 at 8 49 29 AM

2) We don't want the header to get 'cut off' for the trash can column.. we just want the column header text to be blank

Screenshot 2023-06-06 at 8 44 42 AM

3) The trash can doesn't do anything when I click on it, so assuming WIP :)

**Comment by giupb** [comment]
I just fixed the header. Now the header background keeps showing in the trash column. I'm investigating how I can hide the sort option in the trash column as well.

In addition, I tested the trash button functionality, and it seems to be working fine for me. @brifordwylie , can you double-check again?

image

**Comment by luiztauffer** [comment]
@giupb I think what Brian means is a visual effect when clicked, we can probably just simulate a button using CSS on the icon element

**Comment by brifordwylie** [comment]
Okay... so all the stuff above got fixed 👍 

I did notice a couple of issues, so lets not merge until these get resolved

1) update timer (which should update every 5 seconds) isn't updating any more
2) Also looks like the tables are no longer getting dynamically updated (to reflect the state of AWS)
3) I deleted a datasource 'test_data' but actually a ^different^ data source got deleted (I tested this twice)

File: applications/aws_dashboard/pages/main/layout.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def main_layout(
             dbc.Row(models),
             dbc.Row(html.H3("Endpoints"), style={"padding": "10px 0px 0px 0px"}),
             dbc.Row(endpoints),
-            dcc.Interval(id="main-updater", interval=5000, n_intervals=0),
+            dcc.Interval(id="main-updater", interval=500000, n_intervals=0),
```
Comment (luiztauffer) [inline applications/aws_dashboard/pages/main/layout.py]:
```suggestion
            dcc.Interval(id="main-updater", interval=5000, n_intervals=0),
```

**Pull request merged**